meetings/working-groups/extensions/content-based-naming.md

```diff
@@ -0,0 +1,217 @@
+## Summary
+
+The current metadata strategy for skeleton assemblies relies on ordinals to group skeleton types generated from extension blocks. The use of ordinals causes friction in a few parts of the C# ecosystem: 
```

Comment:
I think the ordinal is used to name nested types representing extension blocks. We are not really grouping them.